build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -103,13 +102,20 @@
     <Copy SourceFiles="@(HackFilesToCopy)"
           DestinationFiles="@(HackFilesToCopy->'$(SdkOutputDirectory)/%(RecursiveDir)%(Filename)%(Extension)')" />
     
-      <!-- Publish DotNet -->
+    <!-- Publish DotNet -->
     <DotNetPublish ToolPath="%(Stage.DotnetDir)"
-                                Output="$(SdkOutputDirectory)"
-                                Configuration="$(Configuration)"
-                                VersionSuffix="$(CommitCount)"
-                                ProjectPath="$(SrcDirectory)/redist/redist.csproj"
-                                MSBuildArgs="/p:GenerateRuntimeConfigurationFiles=true"/>
+                   Output="$(SdkOutputDirectory)"
+                   Configuration="$(Configuration)"
+                   VersionSuffix="$(CommitCount)"
+                   ProjectPath="$(SrcDirectory)/redist/redist.csproj"
+                   MSBuildArgs="/p:GenerateRuntimeConfigurationFiles=true" />
+
+    <!-- Publish DotNet -->
```

Comment:
Publish Roslyn? Personally, I find this comment kind of useless. Just look at the ProjectPath.